cli/cmd/cluster.go

```diff
@@ -549,6 +570,7 @@ var _clusterDownCmd = &cobra.Command{
 		}
 
 		fmt.Printf("\nplease check CloudFormation to ensure that all resources for the %s cluster eventually become successfully deleted: %s\n", accessConfig.ClusterName, clusterstate.CloudFormationURL(accessConfig.ClusterName, accessConfig.Region))
+		fmt.Printf("\nthe cluster's %s bucket has been applied a lifecycle rule to empty its contents later today; make sure to delete the %s bucket once the bucket is emptied via the s3 console: https://s3.console.aws.amazon.com/s3/management/%s", bucketName, bucketName, bucketName)
```

Comment:
fmt.Printf("\na lifecycle rule has been applied to the cluster's %s bucket to empty its contents later today; you can delete the %s bucket via the s3 console once it has been emptied: https://s3.console.aws.amazon.com/s3/management/%s", bucketName, bucketName, bucketName)